aces/Loris
# [NDB_BVL_Instrument] Remove PHP warning when checking diff with null

## Brief summary of changes

- PHP warnings were caused when a key was not set, this is resolved.
- Fix recreate_conflicts after removal of Test_name from flag

#### Testing instructions (if applicable)

1. Run recreate_conflicts and confirm it runs okay

[CCNA OVERRIDE PR](https://github.com/aces/CCNA/pull/7196)
